src/bindings/js/node/include/model_wrap.hpp

```diff
@@ -84,6 +84,14 @@ class ModelWrap : public Napi::ObjectWrap<ModelWrap> {
      */
     Napi::Value is_dynamic(const Napi::CallbackInfo& info);
 
+    /**
+     * @brief Return the number of outputs for this model
+     * @param info Contains information about the environment and passed arguments
+     * This method does not accept any arguments. If arguments are provided it throws Napi::Error
+     * @return number indicating number of outputs for the model
```

Comment:
To replace duplicating of 'number' word
```suggestion
     * @return number indicating the quantity of outputs for the model
```